api/rosetta/errors.go

```diff
@@ -35,13 +35,15 @@ const (
 	txLength         = "transaction identifier has invalid hash filed length"
 	txInvalidOpCount = "transaction operations list has an invalid number of operations"
 
-	networkCheck      = "unable to check network"
-	blockRetrieval    = "unable to retrieve block"
-	balancesRetrieval = "unable to retrieve balances"
-	oldestRetrieval   = "unable to retrieve oldest block"
-	currentRetrieval  = "unable to retrieve current block"
-	txRetrieval       = "unable to retrieve transaction"
-	intentDetermine   = "unable to determine transaction intent"
+	networkCheck              = "unable to check network"
+	blockRetrieval            = "unable to retrieve block"
+	balancesRetrieval         = "unable to retrieve balances"
+	oldestRetrieval           = "unable to retrieve oldest block"
+	currentRetrieval          = "unable to retrieve current block"
+	txRetrieval               = "unable to retrieve transaction"
+	intentDetermine           = "unable to determine transaction intent"
+	txReferenceRetrieval      = "unable to retrieve transaction reference block"
+	txSequenceNumberRetrieval = "unable to retrieve account's sequence number"
```

Comment:
```suggestion
	txSequenceNumberRetrieval = "unable to retrieve account sequence number"
```

To stay consistent with the other messages